cli/blkio.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/alibaba/pouch/apis/types"
+	units "github.com/docker/go-units"
+)
+
+// WeightDevice defines weight device
+type WeightDevice struct {
+	values []*types.WeightDevice
+}
+
+func validateWeightDevice(val string) (*types.WeightDevice, error) {
+	pairs := strings.Split(val, ":")
+	if len(pairs) != 2 {
+		return nil, fmt.Errorf("bad format for weight device: %s", val)
```

Comment:
Could you provide more invalid reasons? @Ace-Tang 
I think here we tell user the invalidness of input, but we have not told them why it is invalid which will increase his confusion and time in finding ways to solve this. Maybe more time for us to support these cases. But if we add the invalid reasons. It may reduce much potential work no matter in developing or supporting. How about the format ` fmt.Errorf("invalid WeightDevice %s: WeightDevice must be in format of blabla:blabla", sysctl)`.

I have to say many places have the same issue. While we need to improve that places. In addition, I think we need to update the cli flag illustration as well, since they are not so readable for users. We code them just in developer's perspectives.